### DISCUSSION TITLE
Áudios em MP3 e OGG em dispositivos móveis

### DISCUSSION THREAD

**Discussion by deppbrazil** [frontendbr/forum#768] [Perguntas e Respostas]
# Fala Galera blz ai? #

Estou fazendo uma animação em html, css e js no qual tb tem áudios... tenho os áudios em formatos mp3 e ogg que são os mais adequados para web...

No desktop rola tranquilo, porém em tablets e smartphones vejo que com evento de `onclick()` o audio demora um pouquinho para dar inicio... e isso sai de sincronia com a animação....

Carrego os audios no html através da tag: 
```html
<audio>
   <source src="audio.mp3" type="audio/mpeg"/>
   <source src="audio.ogg" type="audio/ogg"/>
</audio>
```
E depois dou inicio nos audios com evento de `onclick()`

### Tem alguma forma do processamento em dispositivos ficar mais rapido? Pois fiz teste com os audios em um servidor e local e ambos demora um pouco para comecarem a tocar =s ###

[repo](https://github.com/deppbrazil/audiomp3-web)

**Comment by barraponto** [comment]
A pergunta óbvia é: vc sabe se o áudio já está carregado?
Vários jeitos de checar: https://stackoverflow.com/questions/8059434/how-do-you-check-if-a-html5-audio-element-is-loaded

**Comment by ghost** [comment]
Pode tentar fazer o preload dele:
- https://caniuse.com/#search=preload
- https://www.w3schools.com/tags/att_audio_preload.asp
